lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -359,6 +359,21 @@ public function GetQueueContents(array $params)
                     break;
             }
         }
+
+        if (!empty($catids)) {
+            if (in_array(-1, $catids)) {
+                if (count($catids) > 1) {
+                    $categoriesfilter = ' AND tc.categoryid IN (' . implode(',', $catids) . ') OR tc.categoryid IS NULL';
+                } else {
+                    $categoriesfilter = ' AND tc.categoryid IS NULL';
+                }
+            } else {
```

Comment:
A jeśli $catids jest niepuste i nie jest arrayem? Tu jakieś bzdury mogą powstać.

Reply:
Wydaje mi się, że uwzględniłem wszystkie przypadki. Algorytm jest taki:
- !empty($catids)
- in_array(-1, $catids)
- count($catids) > 1

W przypadku gdy: 
- funkcję wywołujemy z innego modułu niż rtqueueview.php (tutaj robię macierz z tego co dostaję w GET jeśli nią nie jest),
- parametr `$catids` to wartość, która jest nie jest macierzą,
- jest niepusta,
- nie wynosi -1, 
to wleci to w linię 371.